pkg/clients/helm/client.go

```diff
@@ -107,26 +112,100 @@ func NewClient(log logging.Logger, config *rest.Config, namespace string) (Clien
 	}, nil
 }
 
-func (hc *client) pullAndLoadChart(repo, name, version, username, password string) (*chart.Chart, error) {
+func getVersion(chartFileName string) string {
+	versionIndex := strings.LastIndex(chartFileName, "-")
```

Comment:
nit: would be more explicit/readable with `strings.Split(chartFileName, "-")` followed by a `strings.TrimSuffix(strParts[1], ".tgz")`. 
Otherwise some comments could be helpful.